src/main/java/webs/hillel/HibernateSession.java

```diff
@@ -0,0 +1,25 @@
+package webs.hillel;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateSession {
+    private static final SessionFactory sessionFactory = buildSessionFactory();
+
+    private static SessionFactory buildSessionFactory() {
+        try {
+            Configuration configuration = new Configuration();
+            configuration.configure("hibernate.cfg.xml");
+            configuration.addAnnotatedClass(Student.class);
+
+            return configuration.buildSessionFactory();
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new RuntimeException("Failed to build the SessionFactory!");
```

Comment:
I would recommend adding a message to your exception, like this: `RuntimeException(e.getMessage(), e);`